spec/gke/hypervisor/gke_spec.rb

```diff
@@ -0,0 +1,178 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'fakefs/spec_helpers'
```

Comment:
you can create a `.rspec` file in the root directory and add 
```
require 'spec_helper'
```
`require 'fakefs/spec_helpers'`  could be required in `spec_helper.rb` if we're gonna use it in each test file.

and any other rspec config. In this way we won't have to require those in each test file.
https://github.com/rspec/rspec/wiki#rspec